packages/gatsby-remark-images-contentful/src/__tests__/index.js

```diff
@@ -16,18 +16,28 @@ jest.mock(`../utils/`, () => {
 })
 
 jest.mock(`axios`)
+
+let mockedSharpMetadata
 jest.mock(`sharp`, () => () => {
-  return {
-    metadata: jest.fn(() => {
-      return {
-        width: 200,
-        height: 200,
-        density: 75,
-      }
-    }),
-  }
+  return { metadata: mockedSharpMetadata }
```

Comment:
```suggestion
  return { metadata: jest.fn() }
```

Reply:
I changed it closer to how gatsby-plugin-sharp mocks sharp. Please have a look